tests/testWrapper.py

```diff
@@ -0,0 +1,434 @@
+        'description': 'To test the local network. ' +
+                       'Hosts will be referred to by their MAC address.',
+    },
+    'vhosts': {
+        'type': 'string',
+        'name': 'vhosts',
+        'default': '',
+        'mandatory': 0,
+        'description': '',
+    },
+    'vhosts_ip': {
+        'type': 'string',
+        'name': 'vhosts_ip',
+        'default': '',
+        'mandatory': 0,
+        'description': '',
+    },
+}
+
+
+@patch('ospd_openvas.db.OpenvasDB')
+@patch('ospd_openvas.nvticache.NVTICache')
+class TestOspdOpenvas(unittest.TestCase):
+
+    @patch('ospd_openvas.wrapper.subprocess')
+    def test_redis_nvticache_init(self, mock_subproc, mock_nvti, mock_db):
+        mock_subproc.check_call.return_value = True
+        w = DummyWrapper(mock_nvti, mock_db)
+        w.redis_nvticache_init()
+        self.assertEqual(mock_subproc.check_call.call_count, 1)
+
+    @patch('ospd_openvas.wrapper.subprocess')
+    def test_parse_param(self, mock_subproc, mock_nvti, mock_db):
+
+        mock_subproc.check_output.return_value = (
+            'non_simult_ports = 22'.encode())
+        w =  DummyWrapper(mock_nvti, mock_db)
+        w.parse_param()
+        self.assertEqual(mock_subproc.check_output.call_count, 1)
+        self.assertEqual(OSPD_PARAMS, OSPD_PARAMS_OUT)
+
+    def test_load_vts(self, mock_nvti, mock_db):
+        w =  DummyWrapper(mock_nvti, mock_db)
+        w.load_vts()
+        self.maxDiff = None
+        self.assertEqual(w.vts, w.VT)
+
+    def test_get_custom_xml(self, mock_nvti, mock_db):
+        out = ('<required_ports>Services/www, 80</re'
+               'quired_ports><category>3</category><'
+               'excluded_keys>Settings/disable_cgi_s'
+               'canning</excluded_keys><family>Produ'
+               'ct detection</family><filename>manti'
+               's_detect.nasl</filename><timeout>0</'
+               'timeout>')
+        w =  DummyWrapper(mock_nvti, mock_db)
+        res = w.get_custom_vt_as_xml_str(
+            '1.3.6.1.4.1.25623.1.0.100061',
+            w.VT['1.3.6.1.4.1.25623.1.0.100061']['custom'])
+        self.assertEqual(len(res), len(out))
+
+    def test_get_severities_xml(self, mock_nvti, mock_db):
+        w =  DummyWrapper(mock_nvti, mock_db)
+        out = ('<severity type="cvss_base_v2">'
+               'AV:N/AC:L/Au:N/C:N/I:N/A:N</severity>')
+
+        severities = w.VT['1.3.6.1.4.1.25623.1.0.100061'].get('severities')
+        res = w.get_severities_vt_as_xml_str(
+            '1.3.6.1.4.1.25623.1.0.100061', severities)
+
+        self.assertEqual(res, out)
+
+    def test_get_params_xml(self, mock_nvti, mock_db):
+        w =  DummyWrapper(mock_nvti, mock_db)
+        out = ('<vt_param type="checkbox" id="Do not randomize the  order  in  which ports are scanned"><name>Do not randomize the  order  in  which ports are scanned</name><default>no</default></vt_param><vt_param type="entry" id="Data length : "><name>Data length : </name></vt_param>')
```

Comment:
```suggestion
        out = '<vt_param type="checkbox" id="Do not randomize the  order  in  which ports are scanned"><name>Do not randomize the  order  in  which ports are scanned</name><default>no</default></vt_param><vt_param type="entry" id="Data length : "><name>Data length : </name></vt_param>'
```

Reply:
Should be split into several lines too. Strings can be split easily using a backslash like
```python3
a = 'foo ' \
    'bar'
```